<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #BDBDBD;</string>
  </property>
  <widget class="QTextEdit" name="input_url">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>841</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 2px solid black;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'맑은 고딕'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gulim';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_convert_color">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #E6E6E6;
	border: 2px solid black;
}

QPushButton:pressed{
	background: #424242;
	border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>컬러 영상으로 변환하기</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar_convert_color">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 2px solid black;
border-radius: 10px;</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1151</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>흑백 영상 URL 입력</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>1201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>16</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>40</y>
     <width>16</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_46">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar_train">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 2px solid black;
border-radius: 10px;</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_train">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #BDBDBD;
	border: 2px solid black;
}

QPushButton:pressed{
	background: #424242;
	border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>분석하기
( 성별 / 나이 )</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_train_emotion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>240</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #BDBDBD;
	border: 2px solid black;
}

QPushButton:pressed{
	background: #424242;
	border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>분석하기
( 감정 )</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar_train_emotion">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>350</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 2px solid black;
border-radius: 10px;</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>input_url</zorder>
  <zorder>btn_convert_color</zorder>
  <zorder>progress_bar_convert_color</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>label_21</zorder>
  <zorder>label_22</zorder>
  <zorder>label_23</zorder>
  <zorder>label_24</zorder>
  <zorder>label_25</zorder>
  <zorder>label_26</zorder>
  <zorder>label_27</zorder>
  <zorder>label_28</zorder>
  <zorder>label_29</zorder>
  <zorder>label_30</zorder>
  <zorder>label_31</zorder>
  <zorder>label_32</zorder>
  <zorder>label_33</zorder>
  <zorder>label_34</zorder>
  <zorder>label_35</zorder>
  <zorder>label_36</zorder>
  <zorder>label_37</zorder>
  <zorder>label_38</zorder>
  <zorder>label_39</zorder>
  <zorder>label_40</zorder>
  <zorder>label_41</zorder>
  <zorder>label_42</zorder>
  <zorder>label_43</zorder>
  <zorder>label_44</zorder>
  <zorder>label_45</zorder>
  <zorder>label_46</zorder>
  <zorder>progress_bar_train</zorder>
  <zorder>btn_train</zorder>
  <zorder>btn_train_emotion</zorder>
  <zorder>progress_bar_train_emotion</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
